python/lsst/cp/pipe/utils.py

```diff
@@ -688,7 +688,7 @@ def arrangeFlatsByExpId(exposureList, exposureIdList):
     """
     flatsAtExpId = {}
     # sortedExposures = sorted(exposureList, key=lambda exp:
-    # exp.getInfo().getVisitInfo().getExposureId())
+    # exp.info.getVisitInfo().id)
```

Comment:
What's going on here?

Reply:
The comment was written too long, and was split when comment line length linting was enabled (part of DM-30245).  It should probably be removed, as it's no longer useful.